docroot/themes/custom/uids_base/templates/field/field.html.twig

```diff
@@ -69,7 +69,7 @@
       </div>
     {% else %}
         {% for item in items %}
-          <{{ item_tag }}{{ attributes.addClass(classes, 'field__item') }}>{{ item.content }}</{{ item_tag }}>
+          <{{ item_tag }}{{ attributes.addClass(classes, 'field__item') }}>{{ block('field_icon') }}{{ item.content }}</{{ item_tag }}>
```

Comment:
What does this do?

Reply:
It allows us to use the `field_icon` block to be used when the label is hidden. That syntax is necessary when placing a twig `block` statement in a template multiple times.